crates/kotlin-ffi/Cargo.toml

```diff
@@ -8,13 +8,15 @@ name = "uniffi_yttrium"
 crate-type = ["cdylib"]
 
 [build-dependencies]
-uniffi_build = "0.28.1"
+uniffi_build = "0.28.3"
 
 [dependencies]
 yttrium = { path = "../yttrium", features = ["uniffi"] }
-uniffi = { version = "0.28.1", features = ["tokio", "cli"] }
+uniffi = { git = "https://github.com/mozilla/uniffi-rs", features = ["tokio", "cli"] }
```

Comment:
Can we pin all of the uniffi deps to a commit ref with `rev`? https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#choice-of-commit